tests/functional/test_accounts.py

```diff
@@ -0,0 +1,88 @@
+from app import app
+import requests
+from requests.auth import HTTPBasicAuth
+
+url = "http://localhost:5000/"
+
+
+def test_accounts_get():
+    with app.test_client():
+        # Test a valid login
+        json = {'email': 'pepe432@gmail.com', 'password': 'pepe123,.'}
+        login = requests.post(url + "API/login", json)
+        assert login.status_code == 200
+
+        # Test invalid access with nonexistent email
+        token = login.json()
+        auth = HTTPBasicAuth(token['token'], '')
+
+        r = requests.get(url + "API/account/pepe433@gmail.com", auth=auth)
+        assert r.status_code == 404
+        assert r.json() == {'message': 'This email [pepe433@gmail.com] does not exist'}
+
+        # Test invalid access with non-matching usernames between current user and request user
+        json = {'email': 'killer23@gmail.com', 'password': 'magic_p443.'}
+        requests.post(url + "API/login", json)
+        r = requests.get(url + "API/account/killer23@gmail.com", auth=auth)
+        assert r.status_code == 400
+        assert r.json() == {'message': "Bad authorization user"}
+
+        r = requests.get(url + "API/account/pepe432@gmail.com", auth=auth)
+        expected = {
+            'account': {
+                'birthday': None,
+                'email': 'pepe432@gmail.com',
+                'is_admin': 0,
+                'name': None,
+                'surname': None,
+                'username': 'pepeman'
+            }
+        }
+
+        assert r.status_code == 200
+        assert r.json() == expected
+
+
+def test_accounts_post():
+    # Test invalid register with missing email
+    json = "{'username': 'dummyname', 'email': None, 'password': 'dummy12.'}"
```

Comment:
El formato del json es un string...
```suggestion
    json = {'username': 'dummyname', 'email': None, 'password': 'dummy12.'}

Reply:
Por algún motivo no me pillaba el json correctamente sin las comillas y lo dejé así. Ya está corregido.